index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div>
+      <h1 id ="title">Microverse Survey</h1>
+      <p id = "description">Fill with your data</p>
+    </div>
```

Comment:
Your headline looks good! However, it would look better if you wrap it with a `section` tag for the semantics of your project.